actions/rules/action_field_set.go

```diff
@@ -0,0 +1,91 @@
+package rules
+
+import (
+	"context"
+	"encoding/json"
+	"reflect"
+
+	errs "github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/fabric8-services/fabric8-wit/application"
+	"github.com/fabric8-services/fabric8-wit/convert"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+)
+
+// ActionFieldSet takes a configuration JSON object that has field
+// names as the keys and a value as the argument. It updates the
+// given ChangeDetector and sets the Field[key] value to the
+// values given. Note that this only works on WorkItems.
+type ActionFieldSet struct {
+	Db     application.DB
+	Ctx    context.Context
+	UserID *uuid.UUID
+}
+
+// make sure the rule is implementing the interface.
+var _ Action = ActionFieldSet{}
+
+func (act ActionFieldSet) storeWorkItem(wi *workitem.WorkItem) (*workitem.WorkItem, error) {
+	if act.Ctx == nil {
+		return nil, errs.New("Context is nil")
```

Comment:
Please have lowercase error messages. (see also https://github.com/golang/go/wiki/CodeReviewComments#error-strings)

Reply:
This is syntactically wrong. Code review rules don't beat grammar.

Reply:
It is not wrong when embedded into a bigger context: `something went wrong: context is nil`. And since we never know when we're at the outmost error let's try to not be against the Go community as a whole and against us. We've established the lowercase error messages for over two years now and I will not change it and won't allow violation of it. I've given you good reason to change it and proof for why we need it. And for what it's worth `Context is nil` is not a sentence. There's a full stop missing.

Reply:
Yes, and I also only removed the "." because of that shitty "convention". In my world, a sentence begins with an uppercase and ends with a ".". Again, convention does not beat grammar. But anyway, fixed (already).